### PR TITLE
UI5 version upgrade + First draft of HDFS Browser component

### PR DESCRIPTION
**Pull request by lokm01**
Signed-off-by: Jan Scherbaum <kmoj02@gmail.com>

### CODE REVIEW

File: menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala [hunk 1]
```diff
@@ -37,15 +37,15 @@ class HDFSConfig @Autowired()(spark: SparkSession) {
 
   @Value("${za.co.absa.enceladus.menas.hadoop.auth.method}") 
   val authMethod: String = ""
-  @Value("${za.co.absa.enceladus.menas.hadoop.auth.krb5.debug}") 
+  @Value("${za.co.absa.enceladus.menas.hadoop.auth.krb5.debug:}")
```
Comment (GeorgiChochov) [inline menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala]: Why the colons at the end here?

File: menas/ui/components/mappingTable/addMappingTable.fragment.xml [hunk 2]
```diff
@@ -14,50 +14,45 @@
...
+                            <Select id="newMappingTableSchemaVersionSelect" class="menasAddMappingSelect" items="{/currentSchemaVersions}"
+                                showSecondaryValues="true" forceSelection="false" width="auto" selectedKey="{/newMappingTable/schemaVersion}">
+                                <core:ListItem key="{version}" text="{version}"
+                                    additionalText="updated: {path: 'lastUpdated', formatter: 'Formatters.stringDateShortFormatter'}"></core:ListItem>
+                            </Select>
+                            <Button icon="sap-icon://action">
+                            </Button>
+                        </FlexBox>
+                        <Label text="HDFS Path"></Label>
+                        <hdfs:HDFSBrowser id="addMtHDFSBrowser" height="300px" width="100%" busyControl="addMappingTableDialog"
```
Comment (GeorgiChochov) [inline menas/ui/components/mappingTable/addMappingTable.fragment.xml]: We need this in `addDataset.frament.xml`, too.

File: menas/ui/components/mappingTable/mappingTableMain.controller.js [hunk 3]
```diff
@@ -19,489 +19,460 @@ jQuery.sap.require("sap.m.MessageBox");
...
+
+    // this needs to be already rendered for it to work
+    setTimeout(function() {
+      scrollCont.scrollToElement(selected, 500);
+    }, 1000);
+  },
+
+  defaultCancel : function() {
+    // This is a workaround for a bug in the Tree component of 1.56.5
+    // TODO: verify whether this was fixed in the subsequent versions
```
Comment (GeorgiChochov) [inline menas/ui/components/mappingTable/mappingTableMain.controller.js]: Is this still an issue in 1.58.2?

File: menas/ui/components/mappingTable/mappingTableMain.controller.js [hunk 4]
```diff
@@ -19,489 +19,460 @@ jQuery.sap.require("sap.m.MessageBox");
...
+
+    this._addDialog.open();
+  },
+
+  schemaSelect : function(oEv) {
+    var sSchemaId = oEv.getParameter("selectedItem").getKey()
+    SchemaService.getAllSchemaVersions(sSchemaId, sap.ui.getCore().byId("newMappingTableSchemaVersionSelect"))
+  },
+
+  mappingTableSelected : function(oEv) {
```
Comment (GeorgiChochov) [inline menas/ui/components/mappingTable/mappingTableMain.controller.js]: I don't think this method is being used anymore

Comment (GeorgiChochov) [reply menas/ui/components/mappingTable/mappingTableMain.controller.js]: Nevermind...

File: menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala (hunk 1 above)
Comment (lokm01) [reply menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala]: As an optional configuration

File: menas/ui/components/Component.js [hunk 5]
```diff
@@ -58,11 +58,13 @@ sap.ui.define( ["sap/ui/core/UIComponent"], function (UIComponent) {
 					},
 					dataset: {
 						viewName: "components.dataset.datasetMain",
-						viewLevel: 0
+						viewLevel: 0,
```
[the remark this answers is not in the document]
Comment (lokm01) [reply menas/ui/components/Component.js]: Yeah seems to be... strange

File: menas/ui/components/mappingTable/addMappingTable.fragment.xml (hunk 2 above)
Comment (lokm01) [reply menas/ui/components/mappingTable/addMappingTable.fragment.xml]: on it...

File: menas/ui/components/mappingTable/mappingTableMain.controller.js (hunk 3 above)
Comment (lokm01) [reply menas/ui/components/mappingTable/mappingTableMain.controller.js]: yeah sadly it is

File: menas/ui/components/Component.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lokm01) [reply menas/ui/components/Component.js]: It was branched too long ago.. did a proper merge now

File: menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala (hunk 1 above)
Comment (GeorgiChochov) [reply menas/src/main/scala/za/co/absa/enceladus/rest/HDFSConfig.scala]: Ah cool, didn't know that!

**Pull request closed without merging**